src/core/src/any.cpp

```diff
@@ -68,6 +79,33 @@ void Any::Base::read_to(Base& other) const {
     }
 }
 
+bool Any::Base::is_base_type_info(const std::type_info& user_type) const {
+    return contains_type_index(base_type_info(), user_type);
+}
+
+bool Any::Base::is_signed_integral() const {
+    return contains_type_index(std::initializer_list<std::type_index>{typeid(char),
```

Comment:
`char` signedness depends on compiler/platform.

Reply:
Added `char` sign check and add it to correct list.